components/style/properties/longhand/box.mako.rs

```diff
@@ -743,7 +743,7 @@ ${helpers.single_keyword("overflow-x", "visible hidden scroll auto",
         #[derive(Debug, Clone, PartialEq)]
         #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
         pub enum AnimationIterationCount {
```

Comment:
Let's also add a link to https://drafts.csswg.org/css-animations/#animation-iteration-count here, or on top of the parse function.